src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
```

Comment:
2. **Unused Private Method:**
   - The `unusedPrivateMethod()` is declared but never used. It should be removed to keep the code clean and maintainable.